Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -0,0 +1,666 @@
+                              instance=args.get('instance'), type=args.get('type'), username=args.get('username'),
+                              group_id=args.get('group_id'), is_admin=args.get('is_admin'), status=args.get('status'),
+                              is_external=args.get('is_external'))
+    request_data, url_suffix = build_filter_and_url_to_search_with(url_suffix, custom_filter, arguments)
+    users_accounts = client.list_users_accounts(url_suffix, request_data)
+    if users_accounts.get('data'):
+        users_accounts = users_accounts.get('data')
+    if args.get('username'):
+        users_accounts = users_accounts[0]
+    human_readable = users_accounts_to_human_readable(users_accounts, args.get('username'))
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='MicrosoftCloudAppSecurity.UsersAccounts',
+        outputs_key_field='username',
+        outputs=users_accounts
+    )
+
+
+def calculate_fetch_start_time(last_fetch, first_fetch):
+    if last_fetch is None:
+        if not first_fetch:
+            first_fetch = '3 days'
+        first_fetch_time = arg_to_timestamp(first_fetch)
+        return first_fetch_time
+    else:
+        last_fetch = int(last_fetch)
+    latest_created_time = last_fetch
+    return latest_created_time
+
+
+def get_max_result_number(max_results):
+    if not max_results:
+        return DEFAULT_INCIDENT_TO_FETCH
+    return int(max_results)
+
+
+def arrange_alert_by_incident_type(alert):
+    incident_types: Dict[str, Any] = {}
+    for entity in alert['entities']:
+        if not entity['type'] in incident_types.keys():
+            incident_types[entity['type']] = []
+        incident_types[entity['type']].append(entity)
+    alert.update(incident_types)
+    del alert['entities']
+
+
+def arrange_alerts_by_incident_type(alerts):
+    if 'data' in alerts.keys():
+        alerts = alerts['data']
+        for alert in alerts:
+            arrange_alert_by_incident_type(alert)
+    else:
+        arrange_alert_by_incident_type(alerts)
+    return alerts
+
+
+def alerts_to_incidents_and_fetch_start_from(alerts, fetch_start_time):
+    incidents = []
+    for alert in alerts:
+        incident_created_time = (alert['timestamp'])
+        incident_created_datetime = datetime.fromtimestamp(incident_created_time / 1000.0).isoformat()
+        incident_occurred = incident_created_datetime.split('.')
+        incident = {
+            'name': alert['title'],
+            'occurred': incident_occurred[0] + 'Z',
+            'rawJSON': json.dumps(alert)
+        }
+        incidents.append(incident)
+        if incident_created_time > fetch_start_time:
+            fetch_start_time = incident_created_time
+    return incidents, fetch_start_time
+
+
+def fetch_incidents(client, max_results, last_run, first_fetch, filters):
+    max_results = get_max_result_number(max_results)
```

Comment:
let's delete the `get_max_result_number` function and use a one line if statement here